### ISSUE TITLE
🐛 'linter/nursery/noUnknownPseudoClass' with false positive for CSS class with multiple colons

### ISSUE THREAD

**Issue by CHE1RON** [biomejs/biome#4533]
### Environment information

```block
CLI:
  Version:                      1.9.4
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v20.18.0"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "pnpm/9.12.3"

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 false

Workspace:
  Open Documents:               0
```

### What happened?

1. Put this CSS in your code:

```css
::-webkit-scrollbar-thumb:hover {
  background-color: #0004;
}
```

2. `noUnknownPseudoClass` will report `:hover` being an unknown pseudo-class:

<img width="366" alt="Screenshot 2024-11-14 at 07 49 28" src="https://github.com/user-attachments/assets/11ba969c-6a8f-4121-a265-ddf5ae42af6a">

### Expected result

`:hover` is recognized as valid pseudo-class

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by CHE1RON** [comment]
Very nice, thank you very much 🙏

**Comment by fireairforce** [comment]
u are welcome~
